packages/docusaurus-types/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@docusaurus/types",
+  "version": "2.0.0-alpha.24",
+  "description": "Common used typings for Docusaurus packages",
+  "main": "./src/index.js",
+  "types": "./src/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "querystring": "0.2.0"
```

Comment:
some missing deps here. eg: webpack typing

Reply:
Oh! Thank you!